src/eventgrid/azext_eventgrid/_help.py

```diff
@@ -20,13 +20,20 @@
 helps['eventgrid domain create'] = """
 type: command
 short-summary: Create a domain.
+parameters:
+  - name: --inbound-ip-rules
+    short-summary: List of inbound ip rules specifying IP Address in CIDR notation e.g., 10.0.0.0/8 along with corresponding Action to perform based on the match or no match of the IpMask. Possible values include - Allow.
+    long-summary: List of inbound ip rules specifying IP Address in CIDR notation e.g., 10.0.0.0/8 along with corresponding Action to perform based on the match or no match of the IpMask. Possible values include - Allow.
 examples:
   - name: Create a new domain.
     text: az eventgrid domain create -g rg1 --name domain1 -l westus2
   - name: Create a new domain with custom input mappings.
     text: az eventgrid domain create -g rg1 --name domain1 -l westus2 --input-schema customeventschema --input-mapping-fields topic=mytopicField eventType=myEventTypeField --input-mapping-default-values subject=DefaultSubject dataVersion=1.0
   - name: Create a new domain that accepts events published in CloudEvents V1.0 schema and maps a property mytopicfield to the topic name.
     text: az eventgrid domain create -g rg1 --name domain1 -l westus2 --input-schema cloudeventschemav1_0 --input-mapping-fields topic=mytopicfield
+  - name: Create a new domain which allows specific inbound ip rules.
+    text: az eventgrid domain create -g rg1 --name domain1 -l westus2 --allow-traffic-from-all-ips false --inbound-ip-rules 10.0.0.0/8 Allow --inbound-ip-rules 10.2.0.0/8 Allow
```

Comment:
Can --allow-traffic-from-all-ips be inferred by --inbound-ip-rules? Maybe --allow-traffic-from-all-ips is redundant.
If --inbound-ip-rules is specified, it means --allow-traffic-from-all-ips is false. Right?

Reply:
No, it can be overwritten by the customer while testing or deploying. It is intentional to keep it that way